clang/lib/Sema/SemaDeclAttr.cpp

```diff
@@ -5503,6 +5503,9 @@ bool Sema::CheckCallingConvAttr(const ParsedAttr &Attrs, CallingConv &CC,
                                   llvm::Log2_64(ABIVLen) - 5);
     break;
   }
+  case ParsedAttr::AT_SPIRKernel:
```

Comment:
Perhaps a silly request, but please sort this as 'not last'.  It is too easy to miss when after a big block.